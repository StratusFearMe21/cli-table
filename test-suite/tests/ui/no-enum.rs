//! `Table` derive macro is not allowed on enums
use cli_table::Table;

#[derive(Table)]
enum Test {
    Variant1,
    Variant2,
}

fn main () {}
