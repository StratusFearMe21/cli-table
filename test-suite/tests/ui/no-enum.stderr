error: `cli_table` derive macros can only be used on structs
 --> $DIR/no-enum.rs:5:1
  |
5 | / enum Test {
6 | |     Variant1,
7 | |     Variant2,
8 | | }
  | |_^
